tests/unit/job/test_query.py

```diff
@@ -689,6 +690,10 @@ def test_ddl_target_table(self):
         self.assertEqual(job.ddl_target_table.dataset_id, "ddl_ds")
         self.assertEqual(job.ddl_target_table.project, self.PROJECT)
 
+    def test_jobtimeout_ms(self): 
+        self.assertIsInstance(job.timeout_ms, value)
```

Comment:
Let's set the job configuration timeout ms to something before asserting the value.

Reply:
Also, move these tests to where we test `QueryJobConfig`: https://github.com/googleapis/python-bigquery/blob/main/tests/unit/job/test_query_config.py#L20